stu12/functions.js

```diff
@@ -5,35 +5,74 @@
 //
 
 const ex1 = () => {
-    console.log('Exercise 1')
+    
+    const printMessage = (messageE1)=> console.log(messageE1)
+    printMessage("Helloo World!")
+    
 }
 
 const ex2 = () => {
-    console.log('Exercise 2')
+    const result = add(2, 3)
+    console.log(result) 
+
+    function add(arg1,arg2){
```

Comment:
Let's convert all these functions to arrow functions.

Reply:
Done